lib/opencensus/absinthe/phase/schema_push.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Opencensus.Absinthe.Phase.SchemaPush do
+  @moduledoc false
+
+  use Absinthe.Phase
+
+  alias Absinthe.Blueprint
+  alias Absinthe.Phase
+
+  @impl true
+  @spec run(Blueprint.t(), keyword()) :: Phase.result_t()
+  def run(blueprint, opts \\ []) do
+    opts = Keyword.put_new(opts, :child_span, "Blueprint")
+    Opencensus.Absinthe.Phase.Push.run(blueprint, opts)
```

Comment:
Maybe instead of calling this span "Blueprint" it might make sense to name it something else. "Schema" might make sense since that's the phase we're starting at, but is a fairly specific term with a generally understood usage.